docs/submit/osg-flock.md

```diff
@@ -71,12 +71,20 @@ Initial Steps
 To be part of OSG, your access point should be registered with the OSG.
 You will need information like the hostname, and the administrative and security contacts.
 Follow the [general registration instructions](../common/registration.md#new-resources).
-For historical reasons, the service type is `Submit Node`.
+For historical reasons, the service type is `Submit Node`. We also request that you tag
+the resources with `OSPool`. An example of a registration is
+[the osg-vo.isi.edu entry](https://github.com/opensciencegrid/topology/blob/master/topology/University%20of%20Southern%20California/Information%20Sciences%20Institute/ISI.yaml)
 
-### Request to be allowed to flock to OSG
-OSG staff will need to add your access point to the list of hosts that flocked jobs are accepted from,
-and provide a HTCondor IDTOKEN so that your new access point can authenticate with OSG.
-Send email to <help@opensciencegrid.org> with the hostname of your access point and request to be added to the list.
+### Register with COManage 
+The adminstrative contact from the the topology entry needs to register with COManage. 
+Instructions can be found [here](https://opensciencegrid.org/technology/policy/comanage-instructions-user/)
+
+Next is to retrive a token so that the new submit host can authenticate with the Open
+Science Pool manager. Please use your COManage registered and approved identity to
+log into the [OSG Token Registration](https://os-registry.opensciencegrid.org/). Once
+logged in, select `Token on Docker`, and find your registered submit node in the list.
+Follow the instructions (you probably have to do the steps on a host with Docker and as
+root), and once you have the token generated, keep that for later steps.
```

Comment:
We should mention this in the `Before Starting` that they will need access to a host with Docker or Podman